### DISCUSSION TITLE
Serverless plugin updated and integrated directly in NextJS

### DISCUSSION THREAD

**Discussion by AlessandroMaro80** [vercel/next.js#41882] [Ideas]
### Describe the feature you'd like to request

Hi guys
First of all, congratulations for your extraordinary work.
We will have a problem: we use the serverless plugin, but unfortunately it is only compatible with version 11 and now we are already at version 13 of nextjs.
We are missing out on many interesting features :(
One above all the on-demand regeneration of pages (next 12).
Couldn't you have a serverless plugin directly in NextJS?
So that every new major release is immediately compatible?

Thank you very much :)

### Describe the solution you'd like

A serverless plugin directly in NextJS?

### Describe alternatives you've considered

https://github.com/serverless-nextjs/serverless-next.js

**Comment by skymarocchinia** [comment]
Any news guys? :)
Thank you
@rauchg @timneutkens @ @nkzawa @arunoda @tonykovanen @impronunciable

**Reply by sladg** [reply]
Sharing my piece of opinion: https://github.com/sladg/nextjs-lambda
SLS is dying of and deployments on Edge Lambdas are insane.
Considering Vercel offers deployment on it's own, there is not much reason to include other clouds by default out-of-box.
